jlh/WeatherApp/app/src/main/java/com/e_conomic/weatherapp/ui/utils/ToolbarAppCompatActivity.kt

```diff
@@ -11,15 +13,20 @@ import com.e_conomic.weatherapp.ui.activities.SettingsActivity
 import org.jetbrains.anko.startActivity
 import org.jetbrains.anko.toast
 
-interface ToolbarManager {
-    val toolbar: Toolbar
+abstract class ToolbarAppCompatActivity : AppCompatActivity() {
+    abstract val toolbar: Toolbar
 
     var toolbarTitle: String
         get() = toolbar.title.toString()
         set(value) {
             toolbar.title = value
         }
 
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        initToolbar()
```

Comment:
You would call the `setContentView()` method here with the abstract value as a parameter.